boons/boons.yml

```diff
@@ -416,7 +416,7 @@
     When you successfully invoke this boon, you gain information from an object or place within range as follows:
     <ul>
     <li><strong>Power Level 5</strong> - The target can read vague ideas and impressions to learn what took place near the object or place within the past hour.</li>
-    <li><strong>Power Level 6</strong> - The target can see a vivid vision, similar to a recording, of what took place near the object or place within the last hour.</li>
+    <li><strong>Power Level 6</strong> - The target can read vague ideas and impressions to learn what took place near the object or place within the past hour.</li>
```

Comment:
@brianfeister Oops is why it was changed. I don't know what I did. Thanks for reviewing and catching it. :)